tests/Traits/SubEntityReferenceTrait.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_content\Traits;
+
+use Drupal\Tests\oe_content\Behat\Content\Traits\GatherSubEntityContextTrait;
```

Comment:
Don't you think that trait that uses another trait is complex? Might be we should use single trait?

Do we need this trait in Unit tests or it should be placed to Drupal\Tests\oe_content\Behat\Content\Traits for Behat tests only?